config/settings.yml

```diff
@@ -112,6 +112,7 @@ feature:
   # results file notifications requires that my_files be on as well
   results_file_notifications_on: true
   set_statement_search_start_date_on: false
+  skip_statement_emails_on: false
```

Comment:
What would you think of inverting the naming here? `send_statement_emails_on` or more explicitly `send_emails_on_statement_create_on` (I find the `_on` suffix really annoying, but I don't think it's worth the effort to change)? Then everyone except DC is set to `true`. I think that might clarify some logic.